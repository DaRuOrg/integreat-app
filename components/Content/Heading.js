import React from 'react'
import PropTypes from 'prop-types'

import style from './Heading.css'

class Heading extends React.Component {
  static propTypes = {
    title: PropTypes.string.isRequired
  }

  render () {
    return <h1 className={style.title}>{this.props.title}</h1>
  }
}

export default Heading
