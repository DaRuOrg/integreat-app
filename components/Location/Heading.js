import React from 'react'

import LocationBig from './assets/LocationBig.png'
import content from './Heading.css'

class Heading extends React.Component {
  render () {
    return (
      <div>
        <div className="row">
          <img className={content.logo} src={LocationBig}/>
        </div>
        <div className="row">
          <h1 className={content.heading}>Where are you?</h1>
        </div>
      </div>
    )
  }
}

export default Heading
