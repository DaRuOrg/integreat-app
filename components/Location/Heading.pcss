.heading {
  text-align: center;
  vertical-align: middle;

  margin-bottom: 30px
}

.logo {
  display: block;
  margin: auto;
  width: 15%;
}
