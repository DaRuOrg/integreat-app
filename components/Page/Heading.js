import React from 'react'
import PropTypes from 'prop-types'

import style from './Heading.pcss'

class Heading extends React.Component {
  static propTypes = {
    title: PropTypes.string.isRequired
  }

  render () {
    return <h1 className={style.title}>{this.props.title}</h1>
  }
}

export default Heading
