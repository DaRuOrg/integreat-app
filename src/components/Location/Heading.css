.heading {
  margin-bottom: 30px;

  text-align: center;
  vertical-align: middle;
}

.logo {
  display: block;
  width: 15%;
  margin: auto;
}
