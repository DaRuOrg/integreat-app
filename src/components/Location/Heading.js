import React from 'react'

import LocationBig from './assets/LocationBig.png'
import style from './Heading.css'
import { translate } from 'react-i18next'
import Caption from '../Content/Caption'

class Heading extends React.Component {
  render () {
    const {t} = this.props
    return (
      <div>
        <div>
          <img className={style.logo} src={LocationBig}/>
        </div>
        <Caption className={style.caption} title={t('Location:where')}/>
      </div>
    )
  }
}

export default translate('Location')(Heading)
