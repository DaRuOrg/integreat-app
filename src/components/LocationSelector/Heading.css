.heading {
  margin-bottom: 30px;
  text-align: center;
  vertical-align: middle;
}

.logo {
  display: block;
  height: 70px;
  margin: 0 auto;
}
