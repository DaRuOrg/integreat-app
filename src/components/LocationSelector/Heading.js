import React from 'react'

import LocationBig from './assets/LocationBig.svg'
import style from './Heading.css'
import { translate } from 'react-i18next'

class Heading extends React.Component {
  render () {
    const {t} = this.props
    return (
      <div>
        <img className={style.logo} src={LocationBig}/>
        <div>
          <h1 className={style.heading}>{t('Location:where')}</h1>
        </div>
      </div>
    )
  }
}

export default translate('Location')(Heading)
