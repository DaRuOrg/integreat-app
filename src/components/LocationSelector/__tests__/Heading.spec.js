import React from 'react'
import renderer from 'react-test-renderer'

import Heading from '../Heading'

jest.mock('store')
jest.mock('react-i18next')

describe('Heading', () => {
  test('should match snapshot', () => {
    const component = renderer.create(
      <Heading/>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
