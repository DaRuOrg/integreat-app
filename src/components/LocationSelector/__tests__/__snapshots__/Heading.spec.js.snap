// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should match snapshot 1`] = `
<div>
  <img
    className="logo"
    src="test-file-stub"
  />
  <h1
    className="caption"
  >
    Location
  </h1>
</div>
`;
